pkg/flowaggregator/flowaggregator.go

```diff
@@ -324,28 +329,50 @@ func (fa *flowAggregator) initExportingProcess() error {
 		return fmt.Errorf("got error when initializing IPFIX exporting process: %v", err)
 	}
 	fa.exportingProcess = ep
-	// Currently, we send two templates for IPv4 and IPv6 regardless of the IP families supported by cluster
-	fa.templateIDv4 = fa.exportingProcess.NewTemplateID()
-	bytesSent, err := fa.sendTemplateSet(false)
+	// Currently, we send 4 templates for IPv4 and IPv6 regardless of the IP families supported by cluster
```

Comment:
Comment can be enhanced with more details by adding the specific requirement in dual-stack clusters.